modules/server/src/main/ts/bedrock/auto/Tunnel.ts

```diff
@@ -0,0 +1,116 @@
+import { ChildProcess } from 'child_process';
+import * as ExecUtils from '../util/ExecUtils';
+import * as split2 from 'split2';
+import * as crypto from 'node:crypto'; // Not needed with Node 19+
+import { Tunnel as LambdaTunnel } from '@lambdatest/node-tunnel';
+
+export interface Tunnel {
+  url: URL;
+  shutdown: () => Promise<void>;
+}
+
+const loadSSH = (subdomain: string, port: number | string): ExecUtils.ChildAPI => {
+  const exec = 'ssh';
+  const args = ['-R', subdomain + ':80:localhost:' + port, 'sish.osu.tiny.work'];
```

Comment:
Just wondering if `'sish.osu.tiny.work'` should be configurable via the the CLI and not baked in. Bedrock is technically a public repo so we have to be mindful of that and assume I we want to keep everything agnostic

Reply:
Changed to accept everything as args or read them from the env